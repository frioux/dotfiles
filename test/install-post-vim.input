#!/bin/bash

dir='test/,install-post'

mkdir -p "$dir/doc"
touch "$dir/doc/something.txt"

make --always-make >/dev/null \
  "INSTALLATION_DIR=$dir" \
  'TARGETS_STATIC=Makefile README' \
  'vim_script_repos_p=t' \
  install

find "$dir" | sort

rm -rf "$dir"

# __END__
