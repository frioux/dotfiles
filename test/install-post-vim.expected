test/,install-post
test/,install-post/Makefile
test/,install-post/README
test/,install-post/doc
test/,install-post/doc/something.txt
test/,install-post/doc/tags
